lib/rouge/lexers/haxe.rb

```diff
@@ -73,41 +73,41 @@ class Haxe < RegexLexer
       end
 
       def self.keywords
-					@keywords ||= Set.new %w(
-							break case cast catch class continue default do else enum false for
-							function if import interface macro new null override package private
-							public return switch this throw true try untyped while
-					)
-			end
-
-			def self.imports
-					@imports ||= Set.new %w(
-							import using
-					)
-			end
-
-			def self.declarations
-					@declarations ||= Set.new %w(
-							abstract dynamic extern extends implements inline
-							static typedef var
-					)
-			end
-
-			def self.cond_keywords
-					@cond_keywords ||= Set.new %w(
-							if else elseif end
-					)
-			end
-
-			def self.reserved
-					@reserved ||= Set.new %w(
-							super trace inline build autoBuild enum
-					)
-			end
-
-			def self.constants
-					@constants ||= Set.new %w(true false null)
-			end
+          @keywords ||= Set.new %w(
+              break case cast catch class continue default do else enum false for
+              function if import interface macro new null override package private
+              public return switch this throw true try untyped while
+          )
+      end
+
+      def self.imports
+          @imports ||= Set.new %w(
+              import using
+          )
+      end
+
+      def self.declarations
+          @declarations ||= Set.new %w(
+              abstract dynamic extern extends implements inline
+              static typedef var
+          )
+      end
+
+      def self.cond_keywords
+          @cond_keywords ||= Set.new %w(
+              if else elseif end
+          )
+      end
+
+      def self.reserved
+          @reserved ||= Set.new %w(
+              super trace inline build autoBuild enum
+          )
+      end
+
+      def self.constants
+          @constants ||= Set.new %w(true false null)
+      end
```

Comment:
Now we're using spaces, which is right, but Ruby's standard is "2 spaces per indent level".

Thank you for caring about this bit.

Reply:
Damn. Ok, I'll make another fix.
Maybe you should use editorconfig?

Reply:
the build failure, I noted what it was, and made a PR for it: https://github.com/jneen/rouge/pull/892